gcovr/gcov.py

```diff
@@ -375,6 +375,37 @@ def parse_tag_line(
             return True
 
         if line.startswith('function '):
+            # function tags look like:
+            #   function X called 0 returned 0% blocks executed 0%
+            #   function X Y called 0 returned 0% blocks executed 0%
+            #   function X Y Z called 0 returned 0% blocks executed 0%
+            #   function X Y Z ... called 0 returned 0% blocks executed 0%
+            # where the percentage should usually be a count.
+            # we could also use a regex: "^function\s+(.+)\s+called\s+(\d+)\s+"
+
+            fields = line.split()  # e.g. "function X Y Z called 0 returned 0% blocks executed 0%"
+            assert len(fields) >= 9, \
+                "Unclear function tag format (count {}): {}".format(len(fields), line)
+
+            # We take all fields excepting the first and the last 7
+            function_name = ' '.join(fields[1:len(fields) - 7])
+            function_call_count = int(fields[len(fields) - 6])
```

Comment:
The len isn't needed because negative indexes are from the end.
```suggestion
            function_name = ' '.join(fields[1:-7])
            function_call_count = int(fields[-6])
```